lib/assets.js

```diff
@@ -21,20 +21,16 @@ function asset(options = { inline: [] }) {
 }
 
 
-const loadAssetsMap = () => {
+
+const loadAssetsMap = (origin) => {
   assetsMap = {};
 
-  const entryPath = resolve('app/assets/entry.html');
-  const distDir = 'public/dist';
-  const entryDir = dirname(entryPath);
-  const entryName = basename(entryPath);
-  const resultPath = resolve(distDir, entryName);
-  const entryPaths = getAssetPaths(fs.readFileSync(entryPath, 'utf8'));
-  const resultPaths = getAssetPaths(fs.readFileSync(resultPath, 'utf8'));
+  const entryDir = 'app/assets';
+  const map = JSON.parse(fs.readFileSync(resolve('assetsMap.json'), 'utf8'));
 
-  entryPaths.forEach((path, p) => {
+  Object.keys(map).forEach((path) => {
```

Comment:
I feel like this loop step is somehow needless in the new paradigm. Instead of this step, couldn't we generate the map such that we don't need this anymore?